### DISCUSSION TITLE
[@types/telegram-web-app] add support for Bot API 8.0

### DISCUSSION THREAD

**Discussion by JackyGohSG** [DefinitelyTyped/DefinitelyTyped#71194] [Issues with a @types package]
Today, Telegram launched the largest update in the history of mini apps. They get a full-screen mode, subscription plans, home screen shortcuts, sending gifts, sharing media, creating documents, access to geolocation, motion tracker, setting emoji statuses — and more.

Could someone please help update @types/telegram-web-app. Many thanks!

https://core.telegram.org/bots/api

[a comment by an automated account is not shown]

**Comment by JackyGohSG** [comment]
@sidorko saw you from previous telegram-web-app update, would appreciate if you could help us out. Thanks!

**Reply by KnorpelSenf** [reply]
By “helping” you, do you mean that you want to lead the update and other people help out?

**Reply by JackyGohSG** [reply]
@KnorpelSenf I would love to but I can't lead the update as I don't have the expertise to update the package.
